apps/sync/js/adapters/history.js

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
Maybe you can add a comment block with a description of:
* The fields of a record in the Kinto collection, and their meaning (including the `deleted: true` case).
* The fields of a Place object, and their meaning.
* The mapping we do between them.

It seems the only thing is converting visits from format:
`{ data: 1234567890123 }`
to format:
`1234567890`,
right?